robust_intelligence_ai_firewall/manifest.json

```diff
@@ -0,0 +1,61 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "1d208134-9005-4a79-bbc1-445950d1a5c7",
+  "app_id": "robust-intelligence-ai-firewall",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "",
+    "title": "Robust Intelligence AI Firewall",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Robust Intelligence AI Firewall",
+        "image_url": "images/ai-firewall.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Robust Intelligence AI Firewall Results dashboard",
+        "image_url": "images/firewall-dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
```

Comment:
This is missing classifier tags such as: Categories, and submitted data type as seen [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags). Ex: 
    "classifier_tags": [
      "Supported OS::Linux",
      "Supported OS::Windows",
      "Supported OS::macOS"
      "Category::<category>"
      "Offering::<offering>"
      "Queried Data Type::<data type>"
      "Submitted Data Type::<data type>"
    ]

Reply:
Thanks, added some tags. We do not query any data, so did not include any `Queried Data Type::` tags